eng/Version.Details.xml

```diff
@@ -170,9 +170,9 @@
       <Uri>https://github.com/dotnet/emsdk</Uri>
       <Sha>b581f08ff228099e732aa8c8cde53995a5870901</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build" Version="0.1.0-alpha.1.21475.1" CoherentParentDependency="Microsoft.NET.Sdk">
-      <Uri>https://github.com/dotnet/source-build</Uri>
-      <Sha>10d0f7e94aa45889155c312f51cfc01bf326b853</Sha>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-externals" Version="7.0.0-alpha.1.22104.1" CoherentParentDependency="Microsoft.NET.Sdk">
+      <Uri>https://github.com/dotnet/source-build-externals</Uri>
+      <Sha>a28332454bd70964205fb18bf3d0a1146a228b25</Sha>
       <SourceBuild RepoName="source-build" ManagedOnly="true" />
```

Comment:
Is there a reason we are not changing the repo name here (and also `repos/source-build.proj` in the tarball)?

Reply:
This was an oversight on my part and it the reason CI failed.